openedx/core/djangoapps/content_libraries/tests/test_api.py

```diff
@@ -546,11 +548,11 @@ def test_set_library_component_collections(self):
         collection_update_event_receiver = mock.Mock()
         LIBRARY_COLLECTION_UPDATED.connect(collection_update_event_receiver)
         assert not list(self.col2.entities.all())
-        component = api.get_component_from_usage_key(UsageKey.from_string(self.lib2_problem_block["id"]))
-
+        component_key = UsageKey.from_string(self.lib2_problem_block["id"])
+        entity_key = xblock_api.entity_key_from_usage_key(component_key)
```

Comment:
Nit: I kinda prefer this as it was, to keep the logic you copied into `entity_key_from_usage_key` contained within openedx-learning. Can't you leave it as it was and just use:

```
        component = api.get_component_from_usage_key(UsageKey.from_string(self.lib2_problem_block["id"]))
        entity_key = component.publishable_entity.key
```
?

Reply:
Ya, I was uneasy about that too.. don't like duplicating critical logic like that. Reverted with [2e7be75](https://github.com/openedx/edx-platform/pull/36528/commits/2e7be75c8353211dbb9e3ffcae8f49ebdc9acb9b).